variants/namino_bianco/pins_arduino.h

```diff
@@ -0,0 +1,205 @@
+//
+// Copyright (c) 2023 Namino Team, bianco version: 1.0.0 @ 2023-10-04
+//
+//
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#include <stdint.h>
+#include "soc/soc_caps.h"
+
+#define USB_VID 0x303a
+#define USB_PID 0x1001
+
+#define NAMINO_BIANCO_BOARD
+
+#define NUM_DIGITAL_PINS            SOC_GPIO_PIN_COUNT    // GPIO 0..48
+#define NUM_ANALOG_INPUTS           20                    // GPIO 1..20
+#define EXTERNAL_NUM_INTERRUPTS     NUM_DIGITAL_PINS      // All GPIOs
+#define analogInputToDigitalPin(p)  (((p)<NUM_ANALOG_INPUTS)?(analogChannelToDigitalPin(p)):-1)
+#define digitalPinToInterrupt(p)    (((p)<NUM_DIGITAL_PINS)?(p):NOT_AN_INTERRUPT)
+#define digitalPinHasPWM(p)         (p < NUM_DIGITAL_PINS)
```

Comment:
Can you please remove this part? For 3.0.0 this will be automatically set in` Arduino.h` file for all SOCs and it's not needed to be defined there anymore.
For reference PR #8706.